tests/installation/test_installer.py

```diff
@@ -174,8 +174,8 @@ def locker(project_root: Path) -> Locker:
 
 
 @pytest.fixture()
-def env() -> NullEnv:
-    return NullEnv()
+def env(tmp_path: Path) -> NullEnv:
+    return NullEnv(path=Path(tmp_path))
```

Comment:
You can pass `tmp_path` directly now instead of constructing a new `Path` from it.